### PR TITLE
remove connector node from risedev

### PR DESCRIPTION
**Pull request by chenzl25**
I hereby agree to the terms of the [RisingWave Labs, Inc. Contributor License Agreement](https://gist.github.com/TennyZhuang/f00be7f16996ea48effb049aa7be4d66#file-rw_cla).

## What's changed and what's your intention?

- Remove the connector node from risedev, since its functionality has been integrated into compute right now.
- Related Issue https://github.com/risingwavelabs/risingwave/issues/12090

## Checklist

- [ ] I have written necessary rustdoc comments
- [ ] I have added necessary unit tests and integration tests
- [ ] I have added fuzzing tests or opened an issue to track them. (Optional, recommended for new SQL features #7934).
- [ ] My PR contains breaking changes. (If it deprecates some features, please create a tracking issue to remove them in the future).
- [ ] All checks passed in `./risedev check` (or alias, `./risedev c`)
- [ ] My PR changes performance-critical code. (Please run macro/micro-benchmarks and show the results.)

- [ ] My PR contains critical fixes that are necessary to be merged into the latest release. (Please check out the [details](https://github.com/risingwavelabs/risingwave/blob/main/CONTRIBUTING.md))

## Documentation

- [ ] My PR needs documentation updates. (Please use the **Release note** section below to summarize the impact on users)

## Release note

If this PR includes changes that directly affect users or other significant modifications relevant to the community, kindly draft a release note to provide a concise summary of these changes. Please prioritize highlighting the impact these changes will have on users.

</details>

### CODE REVIEW

**Comment by kwannoel** [comment]
Do we always want it to be part of compute-node? Are there any occasions we will want to deploy it separately?

File: src/connector/src/sink/log_store.rs [hunk 1]
```diff
@@ -204,9 +204,11 @@ impl<R: LogReader> LogReader for MonitoredLogReader<R> {
                 self.metrics.log_store_latest_read_epoch.set(*epoch as _);
             }
             if let LogStoreReadItem::StreamChunk { chunk, .. } = item {
+                let cardinality = chunk.cardinality();
+                self.metrics.log_store_read_rows.inc_by(cardinality as _);
                 self.metrics
```
Comment (wenym1) [inline src/connector/src/sink/log_store.rs:209]: The metrics was originally reported only in connector node. So it's better to report it only at the remote sink.

**Comment by chenzl25** [comment]
> Do we always want it to be part of compute-node? Are there any occasions we will want to deploy it separately?

We want to simplify our system architecture, so we won't deploy it separately. The cloud team will also remove it from the k8s later.

**Review by fuyufjh — APPROVED** [review]
The part of removing connector-node LGTM

**Review by kwannoel — APPROVED** [review]
Removing connector-node lgtm, since it is already part of CN.

**Review by StrikeW — COMMENTED** [review]
You may need to also remove `src/risedevtool/src/task/connector_service.rs`

File: src/connector/src/sink/log_store.rs (hunk 1 above)
Comment (chenzl25) [reply src/connector/src/sink/log_store.rs]: Let me open another PR to add this metric.

**Pull request merged**